webdriver-spec.html

```diff
@@ -699,13 +700,37 @@
 <section>
 <h2>Interface</h2>
 
-<pre class=idl>partial interface Navigator {
-	[Unforgeable] readonly attribute boolean webdriver;
-};</pre>
+<p>If the <dfn>webdriver-active flag</dfn> is true,
+ the user agent must support the following partial interfaces:
 
-<p data-dfn-for="Navigator">The <dfn><code>webdriver</code></dfn> IDL attribute
- of the <code><a>Navigator</a></code> interface must return
- the value of the <dfn>webdriver-active flag</dfn>, which is initially false.
+ <pre class=idl>
+ partial interface Navigator {
+ };
+ Navigator implements AutomationInformation;
+</pre>
+
+<pre class=idl>
+ [Exposed=Worker]
+ interface WorkerNavigator {};
+ WorkerNavigator implements AutomationInformation;
+</pre>
+
+<section data-dfn-for="AutomationInformation" data-dfn-for="AutomationInformation">
+
+<h2><dfn>AutomationInformation</dfn> interface</h2>
+
+<pre class=idl>
+ [NoInterfaceObject, Exposed=(Window,Worker)]
+ interface AutomationInformation {
+ [Exposed=Window] readonly attribute boolean webdriver; // constant true
```

Comment:
Uh, that's how I interpreted your comments in https://github.com/w3c/webdriver/issues/923 .